### PR TITLE
Make compatible with new flow strict mode

### PR DESCRIPTION
**Pull request by Alxandr**
See #18

### CODE REVIEW

**Comment by Alxandr** [comment]
Note; this might not be enough. Trying to figure out how to test it locally without having to publish to NPM.

**Comment by andreypopp** [comment]
You can add more stuff to `test_flow.js` to test locally.

**Comment by Alxandr** [comment]
This seems to be working (I fixed a small error in the flow_test file). Any chance of it being accepted. If new errors are discovered I can always add a new PR :)

**Comment by andreypopp** [comment]
Hi! Sorry was pretty busy recently. Looking into it.

File: .flowconfig [hunk 1]
```diff
@@ -21,3 +21,4 @@
 esproposal.class_static_fields=enable
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue
 suppress_comment=\\(.\\|\n\\)*\\$ExpectError
+experimental.const_params=true
```
Comment (andreypopp) [inline .flowconfig]: Can we not rely on a global setting? As I understand flow strict mode could be enabled per file.

**Comment by andreypopp** [comment]
Also, `yarn.lock` contained a link to some private URL at artifactory.

**Pull request merged**